SConstruct

```diff
@@ -103,6 +103,28 @@ inst = env['PRJ_SCONS_INSTALL']
 ####new part
 #####PSA. Check for header files and libraries up front
 confinst = Configure(env)
+
+# Try to autodetect numpy
+if confinst.CheckCHeader("numpy/arrayobject.h"):
+    print("Numpy header found.")
+else:
+    print("Numpy not found. Attempting to autodetect...")
+    try:
+        import numpy
```

Comment:
This is tricky because scons could be using a different python version than the one being used for installation - notably macports. Its lot easier with cmake to do this.

Reply:
Yes, I think I have always used the scons from Python2. Not sure why I got started with that, but I never changed it.

Reply:
True. Unless you know how to have scons invoke the python used for installation, maybe I should just disable this when scons is using python 2.